### ISSUE TITLE
Improve performance on some of the slips

### ISSUE THREAD

**Issue by anupamme** [anupamme/ocr_detect#1]
![1](https://user-images.githubusercontent.com/667857/78297814-4b424f80-7528-11ea-8e99-10cff8cea483.png)
![AxisBank-1](https://user-images.githubusercontent.com/667857/78297851-6614c400-7528-11ea-8fa7-108e8c343779.png)
![HDFCBank-1](https://user-images.githubusercontent.com/667857/78297885-7a58c100-7528-11ea-929a-d87521ef826c.png)
![HDFCBank-2](https://user-images.githubusercontent.com/667857/78297899-82186580-7528-11ea-974f-a5ded0b1e655.png)

Current OCR mechanism does not work on these images. So we need a way to re-train the detector and OCR engine for these images.

So basically:

1. given new kinds of documents we need a way to retrain the end to end system.

So that all ML steps are automated and anyone can run them.
